sdk/storage/azure-storage-queue/azure/storage/queue/_encryption.py

```diff
@@ -981,11 +1039,11 @@ def decrypt_queue_message(message, response, require_encryption, key_encryption_
                 'Encryption required, but received message does not contain appropriate metatadata. ' + \
                 'Message was either not encrypted or metadata was incorrect.') from exc
 
-        return message
+        return message_dict
     try:
-        return _decrypt_message(decoded_data, encryption_data, key_encryption_key, resolver).decode('utf-8')
+        return _decrypt_message(decoded_data, encryption_data, key_encryption_key, resolver).decode('utf-8') #type: ignore
     except Exception as error:
         raise HttpResponseError(
             message="Decryption failed.",
-            response=response,
+            response=response, #type: ignore
```

Comment:
`azure\storage\queue\_encryption.py:1048: error: Argument "response" to "HttpResponseError" has incompatible type "PipelineResponse[Any, Any]"; expected "Optional[_HttpResponseCommonAPI]"  [arg-type]`

_HttpResponseCommonAPI is in exceptions.py from core in the build, but not in the copy of core that is in our codebase? Not sure how that works so going to ignore for now.